File: app/src/main/java/com/su/washcall/network/MachineRequest.java
package com.su.washcall.network;

public class MachineRequest {
    private int id;
    private int value;

    public MachineRequest(int id, int value) {
        this.id = id;
        this.value = value;
    }

    public int getId() {
        return id;
    }

    public int getValue() {
        return value;
    }
}

File: app/src/main/java/com/su/washcall/network/washmachinRequest/MachineRequest.kt
package com.su.washcall.network.washmachinRequest

import com.google.gson.annotations.SerializedName

/**
 * 서버로 세탁기 정보를 등록/요청할 때 사용하는 데이터 클래스
 */
data class LoadDataRequest(
    @SerializedName("access_token")
    val accessToken: String
)